shell/store/prefs.js

```diff
@@ -399,6 +414,17 @@ export const actions = {
       return;
     }
 
+    if (Object.keys(prefsBeforeLogin).length > 0) {
```

Comment:
@nwmac How do you ensure that the change preference is called after the login?
What if the user refresh the page after setting the preference?